test/1.4/methods/T_independent.dml

```diff
@@ -0,0 +1,109 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+device test;
+import "testing.dml";
+
+extern int sprintf(char *str, const char *format, ...);
```

Comment:
You can reduce pain a bit by using `strbuf_t`, which is already DML-wrapped. (or you could abandon strings and just use integers everywhere, like `return default() + 100;`)

Reply:
I don't see the need; I'll keep this as-is.